notebooks/00_quick_start/run_notebook_on_aml.ipynb

```diff
@@ -14,121 +14,93 @@
    "metadata": {},
    "source": [
     "# Run Notebook on AML\n",
-    "This noteboook provides an example of how to directly submit notebook to AzureML compute targets. \n",
+    "## Introduction to Azure Machine Learning  \n",
+    "The **[Azure Machine Learning service (AzureML)](https://docs.microsoft.com/azure/machine-learning/service/overview-what-is-azure-ml)** provides a cloud-based environment you can use to prep data, train, test, deploy, manage, and track machine learning models. By using Azure Machine Learning service, you can start training on your local machine and then scale out to the cloud. With many available compute targets, like [Azure Machine Learning Compute](https://docs.microsoft.com/en-us/azure/machine-learning/service/how-to-set-up-training-targets#amlcompute) and [Azure Databricks](https://docs.microsoft.com/en-us/azure/azure-databricks/what-is-azure-databricks), and with [advanced hyperparameter tuning services](https://docs.microsoft.com/en-us/azure/machine-learning/service/how-to-tune-hyperparameters), you can build better models faster by using the power of the cloud.\n",
     "\n",
-    "# 0 Setup environment\n",
+    "Data scientists and AI developers use the main [Azure Machine Learning Python SDK](https://docs.microsoft.com/en-us/python/api/overview/azure/ml/intro?view=azure-ml-py) to build and run machine learning workflows with the Azure Machine Learning service. You can interact with the service in any Python environment, including Jupyter Notebooks or your favorite Python IDE. The Azure Machine Learning SDK allows you the choice of using local or cloud compute resources, while managing and maintaining the complete data science workflow from the cloud.\n",
+    "![AzureML Workflow](https://docs.microsoft.com/en-us/azure/machine-learning/service/media/overview-what-is-azure-ml/aml.png)\n",
     "\n",
-    "### Install azure.contrib.notebook package"
+    "This noteboook provides a scaffold of how to directly submit an existing notebook to AzureML compute targets. \n",
```

Comment:
This needs to be expanded on and updated to show why this is beneficial. Now a user doesn't have to rewrite the training script and submit it directly

Reply:
Added "Now a user doesn't have to rewrite the training script, instead, just by replacing file name, now user can submit notebook directly."